### DISCUSSION TITLE
Enabling feature flag from `pip install`?

### DISCUSSION THREAD

**Discussion by gauteh** [PyO3/maturin#1090] [Q&A] [answered]
Hi,

I'm trying to package a maturin project (mixed python/rust) for conda. I usually have all deps statically compiled for the python package, so the `static` feature is enabled in pyproject.toml which allows me to create manylinux compatible wheels. However, conda-forge doesn't like statically linked libraries: is there any way I can pass an argument to `pip install  .` so that the they are passed on to the maturin build backend?

https://github.com/gauteh/roaring-landmask

Regards, Gaute

**Comment by messense** [comment, marked as answer]
You can set `MATURIN_PEP517_ARGS` env var, for example: `MATURIN_PEP517_ARGS="--features static"`.

https://github.com/PyO3/maturin/blob/c6bbccecb34d5a1e41edd02bc77a99569d4b7dcf/maturin/__init__.py#L34-L36
https://github.com/PyO3/maturin/blob/c6bbccecb34d5a1e41edd02bc77a99569d4b7dcf/maturin/__init__.py#L67-L69

**Reply by gauteh** [reply]
Thanks a lot! I will try that.

**Reply by gauteh** [reply]
That seems to do the trick! Thanks a lot!
